packages/automerge-repo/src/types.ts

```diff
@@ -1,17 +1,22 @@
 /** The ID of a document. Typically you should use a {@link AutomergeUrl} instead.
  */
 export type DocumentId = string & { __documentId: true } // for logging
-/** A branded string representing a URL for a document 
+
+/** A branded string representing a URL for a document
  *
  * @remarks
- * An automerge URL has the form `automerge:<base58 encoded string>`. This 
+ * An automerge URL has the form `automerge:<base58 encoded string>`. This
  * type is returned from various routines which validate a url.
  *
  */
 export type AutomergeUrl = string & { __documentUrl: true } // for opening / linking
+
 /** A document ID as a Uint8Array instead of a bas58 encoded string. Typically you should use a {@link AutomergeUrl} instead.
  */
 export type BinaryDocumentId = Uint8Array & { __binaryDocumentId: true } // for storing / syncing
 
 /** A branded type for peer IDs */
-export type PeerId = string & { __peerId: false }
+export type PeerId = string & { __peerId: true }
```

Comment:
Not really related, and doesn't really affect anything, but it bugged me that we were using `false` in some places and `true` in others.